config/initializers/assets.rb

```diff
@@ -22,5 +22,5 @@
 # Auto precompile CSS
 css_path = File.join(Rails.root, 'app/assets/stylesheets/')
 Find.find(css_path).each do |file|
-  Rails.application.config.assets.precompile << $1.sub(css_path, '') if file =~ (/^(.*)\.scss$/)
+  Rails.application.config.assets.precompile << $1.sub(css_path, '') if file =~ (/^(.*)\.(scss|css)(\.erb)?$/)
```

Comment:
__[RuboCop]__

Performance/RegexpMatch: Use `match?` instead of `=~` when `MatchData` is not used.


<!-- Comment by [SideCI](https://sideci.com) -->